### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by codewithed**
Completed the following actions for the Portfolio website:

- Create a feature branch according to GitHub flow.
- Setup a GitHub workflow with linters.yml file.
- Add index.html file
- Add styles.css file
- Add .gitignore file
- Update README.md with relevant Information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class="toolbar">
+        <div class="logo-container">My Logo</div>
+        <div></div>
+        <div><button><img src="icon-menu.png" alt="a menu icon"></button></div>
+    </section>
+    <section class="headline-section">
+        <p class="intro">I'm Edmund Glad to see you!</p>
```
Comment (omarsalem7) [inline index.html:17]: - [ ] Consider to use `<h1>` for your title to improve SEO.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <p class="main-text">I’m a software developer! I can help you build a product ,
+         feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+        <div class="connect">LET'S CONNECT</div>
+        <div class="social-media"><img src="social_media_icons.png" alt="some social media icons"></div>
```
Comment (omarsalem7) [inline index.html:23]: - [ ] Kindly make sure to extract each icon from figma and the structure will be something like this:
```html
<ul>
   <li><a> <img/> </a></li>
   <li><a> <img/> </a></li>
</ul>
```

**Comment by codewithed** [comment]
Thank you for your clear review @omarsalem7

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @codewithed       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**